packages/common/src/index.ts

```diff
@@ -6,4 +6,5 @@ export * from './crypto'
 export * from './expand'
 export * from './json'
 export * from './logger'
+export * from './logic/sourcemap'
```

Comment:
The `logic` directory feel unneeded, can we lift to the root like Python has it?